app/views/responsible_body/devices/schools/index.html.erb

```diff
@@ -42,7 +42,7 @@
       <th class="govuk-table__header app-schools-table__school-column">School</th>
       <th class="govuk-table__header">Allocation</th>
       <th class="govuk-table__header">Who will order?</th>
-      <th class="govuk-table__header">Status</th>
+      <th class="govuk-table__header app-schools-table__status-column">Status</th>
```

Comment:
Let's use `govuk-table__cell--nowrap` for now

Reply:
We've recently introduced a particularly long status and I'm not sure whether this looks right:

![image](https://user-images.githubusercontent.com/23801/91103410-5165d380-e663-11ea-92ef-e93bb2752890.png)

Of course it may be better to shorten the status name – I'm open to suggestions.

Reply:
"School will be contacted"?